cli/integration_tests/monorepo_one_script_error/run.t

```diff
@@ -52,3 +52,27 @@ Make sure it isn't cached
   
    ERROR  run failed: command  exited (1)
   [1]
+
+Running with --output-mode=errors-only gives error output but no cache hit/miss message
+  $ ${TURBO} --output-logs=errors-only error
+  \xe2\x80\xa2 Packages in scope: my-app (esc)
+  \xe2\x80\xa2 Running error in 1 packages (esc)
+  \xe2\x80\xa2 Remote caching disabled (esc)
+  my-app:error: ERROR: command finished with error: command \(.*/run\.t/apps/my-app\) npm run error exited \(1\) (re)
+  my-app:error: 
+  my-app:error: > error
+  my-app:error: > echo 'intentionally failing' && exit 2
+  my-app:error: 
+  my-app:error: intentionally failing
+  my-app:error: npm ERR! Lifecycle script `error` failed with error: 
+  my-app:error: npm ERR! Error: command failed 
+  my-app:error: npm ERR!   in workspace: my-app 
+  my-app:error: npm ERR!   at location: .*/run.t/apps/my-app  (re)
+  command \(.*/run.t/apps/my-app\) npm run error exited \(1\) (re)
+  
+   Tasks:    0 successful, 1 total
+  Cached:    0 cached, 1 total
+    Time:\s*[\.0-9]+m?s  (re)
+  
+   ERROR  run failed: command  exited (1)
+  [1]
```

Comment:
hmm, I think we should also have a test case where there are two scripts that run, one that errors and one that doesn't. You could add to the `monorepo` fixture in this same integration test or add a totally new `cli/integration_test/monorepo_errors_only_output`, I don't have a strong preference!

Reply:
OK, I added another task to this folder, seemed the simplest.